.yarnrc.yml

```diff
@@ -0,0 +1 @@
+enableScripts: false
```

Comment:
Are both yarnrc's required?

Reply:
I think the .yarnrc.yml is for yarn 2.0 and can be removed. 
The lavamoat cli auto-created these files during setup.